.vscode/launch.json

```diff
@@ -7,7 +7,10 @@
 			"type": "extensionHost",
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
-			"args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+			"args": [
+				"--disable-extensions",
```

Comment:
To enhance the debugging efficiency, the CLI parameter is used to turn off all other VSCode extensions and only the extension "flow-for-vscode" will be run in the [Extension Host] instance.